support-frontend/assets/components/checkoutForm/checkoutForm.jsx

```diff
@@ -15,19 +15,24 @@ type FormSectionPropTypes = {|
   title: Option<string>,
   children: Node,
   headingSize: HeadingSize,
+  noBorder: Boolean,
```

Comment:
this should be `boolean` and then you don't need the additional import

Reply:
Thanks Rupert - I have just fixed it